server/modules/surveyRdb/repository/dataView/read.js

```diff
@@ -162,8 +162,10 @@ export const fetchViewData = async (params, client = db) => {
   _prepareFromClause({ queryBuilder, viewDataNodeDef, nodeDefCols, editMode })
 
   // WHERE clause
-  queryBuilder.where(`${viewDataNodeDef.columnRecordCycle} = $/cycle/`)
-  queryBuilder.addParams({ cycle })
+  if (!R.isNil(cycle)) {
```

Comment:
what happens if you have data in several cycles? will you have "duplicate" records in the export? should we pass the cycle as parameter when starting the export?

Reply:
nope.  a record can only belong to a cycle. isn't it?